tests/CMakeLists.txt

```diff
@@ -195,6 +195,8 @@ if (LIBXML2_FOUND)
         include_directories(${LIBXML2_INCLUDE_DIR})
 	include_directories(externals/Catch/include ${COMMON_INCLUDES})
 	add_executable(test-asebahttp test-http.cpp)
+	# Use set_target_properties instead of target_compile_options to make CMake <= 2.8.11 happy
+	set_target_properties(test-asebahttp PROPERTIES COMPILE_FLAGS -std=c++11)
```

Comment:
Yeah. Also, it looks like [there is a way](https://github.com/philsquared/Catch/blob/master/docs/configuration.md#c-conformance-toggles) to tell Catch that the compiler doesn't support C++11. I'm thinking it's better to do that so that aseba is buildable by more compilers. If you disagree, let me know. Otherwise I'll try making that change later tonight.